.github/workflows/deploy.yml

```
@@ -0,0 +1,40 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches: ['main', 'master']
+  workflow_dispatch:
+    inputs:
+      deploy-msg:
+        description: 'Deploy Message (visible in Netlify)'
+        required: true
+
+jobs:
+  deploy:
+    name: Build and Deploy to Netlify
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
```

```suggestion
      - name: Checkout
        uses: actions/checkout@v3
```
Should we use the latest one?